go.mod

```diff
@@ -1,27 +1,37 @@
 module github.com/dgraph-io/badger/v3
 
-go 1.12
-
-// replace github.com/dgraph-io/ristretto => /home/amanbansal/go/src/github.com/dgraph-io/ristretto
```

Comment:
Let's remove the specific person's and keep the comment

Reply:
Will add a reference in another PR. Thanks for taking a look.